lib/charCounter.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+/* eslint-disable */
+
+/**
+ * Function to count how many characters are remaining in text fields
+ *
+ * @param {Object} input - inputs in this plugin's instance and their values as {String}
+ * *
+ * @module charCounter
+ */
+
+module.exports = function charCounter(input) {
+  var msg = input.parent.querySelector('.input-plugin-textarea'),
+      charRemain = input.parent.querySelector('.feedback-number'),
+      maxChar = input.parent.querySelector('.input-plugin-textarea').maxLength,
+      charCount = function () {
```

Comment:
* `e.target` is your friend here to make sure the right thing is being updated.
* Don't use `innerHTML`, it's very dangerous. Use `textContent`